src/DataManagement/Saving/AlternativeSavingCalculator.cs

```diff
@@ -57,7 +70,6 @@ public CalculationResults GetSuggestedExpensesOffers(IEntryList entryList, IGoal
                             break;
                         default:
                             throw new Exception("Unexpected saving type");
-                            break;
                     };
```

Comment:
Unnecessary semicolon here

Reply:
Arnas, for that minimal not showing everything, it's a feature, it purposefully returns an empty list (a little bit of front-end notification is missing here). Because minimal saving only suggests you to lower "unnecessary" and "low" importances, i believe, so in a case where you only spent money on, for example, "high" importance expenses, theres no way to suggest you to save more in a "minimal" way.